README.md

```diff
@@ -1,14 +1,14 @@
-# tinycolor
+# tinycolor2
 > TinyColor is a small library for Flutter color manipulation and conversion
 
 A port of [tinycolor2](https://github.com/bgrins/TinyColor) by [Brian Grinstead](https://github.com/bgrins)
```

Comment:
Outdated url?

Reply:
Which one. They work fine for me.

Reply:
First one should be this repo url? You can mention the original repo.

Reply:
Why? It's referencing the original repo https://github.com/bgrins/TinyColor and author https://github.com/bgrins, which is correct in my opinion.
We could rather additionally credit the repo/authors we forked this from!?